lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -1023,6 +1023,7 @@ public static function paytypesFunction(array $params, $template)
         }
 
         $elemname = $params['elemname'];
+        $id = $params['id'];
```

Comment:
Czy `id` jest obowiązkowe? Jeśli nie to co tu się wydarzy, gdy `$params['id']` nie będzie zdefiniowane? Nie będzie czasem ostrzeżenia PHP?